### DISCUSSION TITLE
Digit different colours

### DISCUSSION THREAD

**Discussion by Dnoam1959** [cagnulein/qdomyos-zwift#1546] [General]
Where can I find out about the display different digits numbers ? Orang? Green?
Thanks.

**Comment by cagnulein** [comment]
@Dnoam1959 here you can find the colors for the speed https://github.com/cagnulein/qdomyos-zwift/blob/20027b914ef5f148d41dc3f0682e90390c435bc4/src/homeform.cpp#L3537
